### PR TITLE
Gml 1378 oo schema docs

### PR DESCRIPTION
**Pull request by parkererickson-tg**

### CODE REVIEW

File: pyTigerGraph/schema.py [hunk 1]
```diff
@@ -153,7 +276,29 @@ def __repr__(self):
...
+
+    ```
+    @dataclass
+    class HOLDS_ACCOUNT(Edge):
+        opened_on: datetime
+        from_vertex: Union[AccountHolder, g.vertex_types["Account"]]
+        to_vertex: g.vertex_types["Account"]
+        is_directed: bool = True
+        reverse_edge: str = "ACCOUNT_HELD_BY"
+        discriminator: str = "opened_on"
```
Comment (billshitg) [inline pyTigerGraph/schema.py:296]: Sorry I forgot: do we allow the use of multiple attributes as discriminator?

**Review by billshitg — APPROVED** [review]

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: SUCCESS, e2e Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/960/

File: pyTigerGraph/schema.py [hunk 2]
```diff
@@ -1,3 +1,80 @@
...
+conn = TigerGraphConnection("http://YOUR_HOSTNAME_HERE")
+
+g = Graph()
+
+@dataclass
+class Paper(Vertex):
+    id: int
+    y: int
+    x: List[int]
+    primary_id: str = "id"
```
[the remark this answers is not in the document]
Comment (parkererickson-tg) [reply pyTigerGraph/schema.py:61]: Correct. Will denote that.

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: SUCCESS, e2e Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/962/

**Review by qe-tigergraph — CHANGES_REQUESTED** [review]
Unit Test: SUCCESS, e2e Test: FAILURE, Jenkins_job:http://192.168.99.101:30080/job/mlwb_build/967/

**Pull request merged**